src/openrct2/ride/Ride.cpp

```diff
@@ -4917,9 +4917,14 @@ uint64_t ride_entry_get_supported_track_pieces(const rct_ride_entry* rideEntry)
           SpritePrecision::Sprites4 }, // TRACK_INLINE_TWIST_INVERTED
         { SpriteGroupType::Slopes25, SpritePrecision::Sprites4, SpriteGroupType::Slopes60, SpritePrecision::Sprites4,
           SpriteGroupType::Slopes75, SpritePrecision::Sprites4, SpriteGroupType::Slopes90,
-          SpritePrecision::Sprites4 }, // TRACK_QUARTER_LOOP_UNINVERTED
+          SpritePrecision::Sprites4 }, // TRACK_QUARTER_LOOP_UNINVERTED_UP
```

Comment:
You made a mistake somewhere in this function. This causes the available sprites for the Rocket Cars not to be detected correctly, which in turn causes the Classic Mini RC in bpb.sv6 to be imported as a Junior RC, which in turns fails the replay test.

Reply:
I saw that but I can't recreate it, they import properly for me insofar as the object selection is concerned:

![image](https://user-images.githubusercontent.com/3454156/182955545-94b7c40a-f2f7-428d-ad57-4b674c8c5624.png)
![image](https://user-images.githubusercontent.com/3454156/182955667-840447ac-91f5-4342-b2c4-0ad7bf0bc91c.png)

It is puzzling because the check that fails is testing the 10th element and my changes are all the way at the bottom.

Reply:
I just tested Blackpool Pleasure Beach.SC6 and it imported as a mini roller coaster, not a classic/junior coaster.

Reply:
Note that I said bpb.sv6. That is the name of the test file used.

Reply:
I believe the issue is the bit shift wraps around. I'm working on a fix.

Reply:
Well spotted!